homeassistant/components/sun/config_flow.py

```diff
@@ -0,0 +1,33 @@
+"""Config flow to configure the Sun integration."""
+from __future__ import annotations
+
+from typing import Any
+
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigFlow
+from homeassistant.data_entry_flow import FlowResult
+
+from .const import DEFAULT_NAME, DOMAIN
+
+
+class SunConfigFlow(ConfigFlow, domain=DOMAIN):
+    """Config flow for Sun."""
+
+    VERSION = 1
+
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
+        """Handle a flow initialized by the user."""
+        if self._async_current_entries():
+            return self.async_abort(reason="single_instance_allowed")
+
+        if user_input is not None:
+            return self.async_create_entry(title=DEFAULT_NAME, data={})
+
+        return self.async_show_form(step_id="user", data_schema=vol.Schema({}))
```

Comment:
This is no longer required :)

```suggestion
        return self.async_show_form(step_id="user")
```